modules/programs/nushell.nix

```
@@ -145,11 +145,24 @@ in {
     };
 
     environmentVariables = mkOption {
-      type = types.attrsOf types.str;
+      type = types.attrsOf hm.types.nushellValue;
       default = { };
-      example = { FOO = "BAR"; };
+      example = literalExpression ''
+        {
+          FOO = "BAR";
+          LIST_VALUE = [ "foo" "bar" ];
+          NU_LIB_DIRS = lib.concatStringsSep ":" [ ./scripts ];
+          PROMPT_COMMAND = lib.hm.nushell.mkNushellInline "{|| \"> \"}";
```

It is reasonably idiomatic to use `''` strings if they help avoid escaping:

```suggestion
          PROMPT_COMMAND = lib.hm.nushell.mkNushellInline ''{|| "> "}'';
```

---

Had to use `'''` for escaping, but agree